optaplanner-test/pom.xml

```diff
@@ -41,8 +41,8 @@
     </dependency>
     <!-- Testing -->
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
```

Comment:
Technically, this is a backwards incompatible change. Any reason?

Reply:
Good point. This can't be merged on 7.x

Reply:
And needs an upgrade recipe line.